web/js/FlexyConnector.js

```diff
@@ -251,6 +252,8 @@ function loadPage()
 
       var options = ["true", "false"];
       addCellsSelection("Post Duplicate Tag Values:", obj.AppConfig.PostDuplicateTagValues, "PostDuplicateTagValues", options);
+      options = ["omf", "piwebapi"];

```

Comment:
didn't we decide to change these to more user friendly values?  PiWebAPI 2019 or newer or Legacy, PiWebAPI 2018 or older. If this changes the webpage image will need to get changed.